src/services/boardService.js

```diff
@@ -0,0 +1,12 @@
+
+function makeBoard(ranks, files) {
+  return ranks.reduce((board, rank) => {
+    files.forEach(file => {
```

Comment:
I heard what you were saying yesterday, and now I see it. Flip the files and ranks here and you'll have the letters all the same down the columns and the numbers all the same across the rows.